crates/ibc/src/core/ics02_client/handler/update_client.rs

```diff
@@ -329,7 +329,9 @@ mod tests {
             let client_state = {
                 #[allow(deprecated)]
                 let raw_client_state = RawTmClientState {
-                    chain_id: ChainId::from(tm_block.header().chain_id.clone()).to_string(),
+                    chain_id: ChainId::from_str(tm_block.header().chain_id.to_string().as_str())
```

Comment:
Here we can go away with: `ChainId::from(...)`

Reply:
have fixed!